src/problems/starPattern.js

```diff
@@ -3,22 +3,20 @@ for a given value of n, print a star pattern starting from 1 star each line to n
 */
 
```

Comment:
@Reynadi531 Please do not remove the previous solution. You can add yours below it.

Reply:
Ok i'll make a new PR

Reply:
Done on my master commit